### DISCUSSION TITLE
Add ability to run initial bash script based on user or group

### DISCUSSION THREAD

**Discussion by costrouc** [nebari-dev/nebari#690] [Ideas]
This would allow us to initialize user directories

**Comment by danlester** [comment]
Related to #521 https://github.com/Quansight/qhub/pull/521 which should give infrastructure for this at least, as long as there is a .profile script or something that is automatically run. Or maybe we would still need to specify a file that should be run for each user on creation of their container for the first time.

**Comment by costrouc** [comment]
This needs to be better scoped out. How exactly do we plan to specify the script that runs for a specific group in the qhub-config.yaml? Or would this be something that is dropped into `.userskel`

**Reply by viniciusdc** [reply]
Hi @costrouc, do we have an issue for this? or have we solved this?

**Comment by Adam-D-Lewis** [comment]
I'm closing old discussions in a repo cleanup effort.  Feel free to re-open if needed.
